public/index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico" />
```

Comment:
Dit was een ico voor backward compatiblity denk ik. Fancy icons gaan via de manifest. Denk ik wel redelijk om even een icon via 1 of andere online tool te maken en die te gebruiken.